app/views/unattended/provisioning_templates/cloud_init/cloud_init_default.erb

```diff
@@ -20,6 +20,8 @@ description: |
   booted from the image. The image must have cloud-init installed in order for this to work.
   This template accepts the following parameters:
   - force-puppet: boolean (default=false)
+  - enable-openvox8-repo: boolean (default=false)
+  - enable-openvox7-repo: boolean (default=false)
```

Comment:
openvox7 and all the puppetlabs repos aren't actively maintained anymore. Does it make sense to first drop them, and then add openvox 8?